lib/board.ex

```diff
@@ -27,4 +27,11 @@ defmodule Board do
     Enum.filter(board, fn(space) -> space != "X" and space != "O" end)
   end
 
+  def available_spaces_number(board) do
+    length(available_spaces(board))
+  end
```

Comment:
Personally, I think `length(available_paces(board))` is just as clear as giving it a new name. You could also reorder your functions like
```
board
|> available_spaces
|> length
``` 
if that is more clear.